examples/mobilenet/mobilenet_qnn_ep_template.json

```diff
@@ -14,37 +14,46 @@
             }
         }
     },
+    "data_configs": [
+        {
+            "name": "accuracy_data_config",
```

Comment:
since these two datasets are the same, I think it's okay to just use this data config for accuracy and latency. the latency metric will just ignore the post processing component.

Reply:
Done.